tools/upgrade_versions.py

```diff
@@ -114,13 +114,17 @@ def update(self, user_name, user_email):
 
     def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
         LOG.info("%s: Bumping release" % self.pkg)
+        if log is None:
+            log = []
 
-        if self.pkg.commit_id:
+        old_commit_id = self.pkg.sources[0]["git"]["commit_id"]
+        if old_commit_id:
+            new_commit_id = pkg.sources[0]["git"]["commit_id"]
```

Comment:
I'd prefer to put pkg.sources[0]["git"] in a new_source variable instead of duplicating it here and below, but up to you